mlflow/tracking/__init__.py

```diff
@@ -353,11 +352,17 @@ def _get_legacy_artifact_repo(file_store, run_info):
 
 
 def _get_git_commit(path):
```

Comment:
Looks good, would be nice to add a unit test in `tests/tracking/test_tracking.py` for the case where `git` isn't on the PATH - for example we test the desired behavior in `projects.run` when `conda` isn't installed [here](https://github.com/databricks/mlflow/blob/07ffae5106b4c65958892358e26caeeaeec2bdb1/tests/projects/test_projects.py#L68)

Reply:
For some reason, changing the PATH in the test itself doesn't cause an appropriate error to be thrown. Instead, gitpython successfully finds git nevertheless. I am not sure it's worth debugging at this time.